client/my-sites/marketplace/pages/submission-success/signup-success.tsx

```diff
@@ -1,10 +1,12 @@
-import { Gridicon } from '@automattic/components';
 import { ThemeProvider, Global, css } from '@emotion/react';
 import { Button } from '@wordpress/components';
 import { useTranslate } from 'i18n-calypso';
+import checkCircleImage from 'calypso/assets/images/marketplace/check-circle.png';
+import settingsImage from 'calypso/assets/images/marketplace/settings.png';
+import shoppingCartImage from 'calypso/assets/images/marketplace/shopping-cart.png';
```

Comment:
@lupus2k Should we use svgs for these images? I've seen more svgs than pngs in Calypso.

Reply:
Sounds good, I will do it

Reply:
Done